ompi/mca/fcoll/dynamic/fcoll_dynamic_file_write_all.c

```diff
@@ -51,7 +51,7 @@ static int local_heap_sort (mca_io_ompio_local_io_array *io_array,
 
 
 int
-mca_fcoll_dynamic_file_write_all (ompio_file_t *fh,
+mca_fcoll_dynamic_file_write_all (struct ompio_file_t *fh,
```

Comment:
Why did all of these need to be changed to `struct ompio_file_t`?

Reply:
the prototypes in fcoll.h have the struct ompio_file_t syntax, and I just wanted to be consistent